src/services/__tests__/QueueService.int.spec.ts

```diff
@@ -73,9 +79,21 @@ describe("QueueService tests", () => {
         expect(resultJoin).toHaveLength(1);
         expect(receivedQueueTime).toEqual(expectedQueueTime);
       });
-    });
+      it("In Active Match | Does not add to queue", async () => {
+        const mockMatchId = faker.datatype.uuid();
+        const mockPlayers = ActiveMatchBuilder.many(6, { matchId: mockMatchId });
+        await manuallyAddActiveMatch(mockPlayers);
+
+        const oneOfThePlayers = faker.random.arrayElement(mockPlayers);
+        mocked(ActiveMatchRepository.isPlayerInActiveMatch).mockResolvedValue(true);
+
+        const resultJoin = await joinQueue(oneOfThePlayers.id, "MockName");
 
+        expect(resultJoin.find((id) => id)).not.toBe(mockPlayer1.id);
```

Comment:
I think what you want here is a `expect(resultJoin.include(mockPlayer1.id)).toBeFalsey()`, right?

Reply:
Yes, that's a better way to write it.  Fixed in next commit.